config.yaml

```diff
@@ -12,6 +12,75 @@ ignoreFiles:
 taxonomies:
   tag: ""
   category: ""
+menu:
+  main:
+    - identifier: features
+      name: Features
+      url: http://localhost:1313/wiki/index.php/OpenEMR_Features
```

Comment:
seeing several places where using localhost:1313 that should prob just be /wiki/... or /chat

Reply:
FIxed